docs/server/project_setup.rst

```diff
@@ -0,0 +1,175 @@
+:file:`translations` directory within Pootle codebase.
+
+As told above, we will be using a GNU layout for the examples. A GNU layout
+means that a given project contains only translation files named by its
+language code. That means that within that project there are no directories,
+just files. This is the simplest layout possible. Below you can see an example
+with two projects using the GNU layout:
+
+.. code-block:: bash
+
+    `-- translations
+        `-- project1
+        |   |-- de.po
+        |   |-- fr.po
+        |   |-- gl.po
+        |   |-- pt_BR.po
+        |   `-- templates.pot
+        `-- project2
+            |-- af.po
+            |-- eu.po
+            |-- pt_BR.po
+            |-- templates.pot
+            `-- zu.po
+
+You might have noticed that among the regular translation files there are two
+files named :file:`templates.pot`. These are the master files that contain the
+original strings. Usually these master files contain English strings, but we
+find much less confusing to use ``templates`` instead of ``en`` (language code
+for English).
+
+Create a :file:`my-project` directory within
+:setting:`POOTLE_TRANSLATION_DIRECTORY` and place on it the translation files
+for your new project. Make sure you have a :file:`templates.pot` among those
+project translation files.
+
+
+.. _project_setup#create-new-project:
+
+Creating the project
+++++++++++++++++++++
+
+In the top right menu of the interface (or top left if you are using an RTL
+interface), you should see your newly created administrator user. Click on it
+and from the drop down menu, and then click on **Admin** (highlighted in red):
+
+.. image:: ../_static/accessing_admin_interface.png
+
+
+Now you are in the administration interface. Within the **Projects** tab you
+will see a button called **New Project** on the top right (top left in RTL
+interfaces), just below the main menu that has your administrator username:
+
+.. image:: ../_static/add_project_button.png
+
+
+Click on that button and the **Add Project** form will be presented to you.
+Fill it with the new project details. **Code** must match the name for the
+directory within :setting:`POOTLE_TRANSLATION_DIRECTORY` that contains the new
+project translation files, in our example :file:`my-project`. You can also
+provide a **Full Name** easily readable for humans. You don't need to change
+the rest of the fields unless you need to further customize your project.
+
+.. image:: ../_static/add_project_form.png
+
+
+Once you are done click on the *Save* button below the form to create the
+project. Creating the project doesn't actually import all the translations to
+Pootle, so you also need to run :djadmin:`update_stores`:
+
+.. code-block:: bash
+
+    pootle update_stores --project=my-project
+
+
+This will import all the translations from disk into Pootle, calculate the
+translation statistics and calculate the quality check failures. This might
```

Comment:
Added this as one of the assumptions, along with having Pootle up and running and being logged as some admin user.